app/routes/requests/scan.ts

```diff
@@ -8,12 +9,10 @@ type ScanArguments = {
     "min_sales": number, // High quality
     "hq": boolean, // Home server
     "home_sever": string, // Filters, see [docs]
-    "filters": number, // Same ... data center? only
+    "filters": list, // Same ... data center? only
```

Comment:
There is no concept of a `list` within Javascript.

You'd be looking to either format it in either of these ways:

`"filters": number[],`
or 
`"filters": Array<number>,`
Both work identically.

Reply:
See https://www.typescriptlang.org/docs/handbook/basic-types.html#array